test.html

```diff
@@ -106,7 +106,7 @@
 					ctx.eventRightclick(event, jsEvent, view);
 					return false;
 				},
-				defaultDate: '2015-02-12',
+				defaultDate: '2015-02-1',
```

Comment:
The tests fail in FF because of this line. Should be `2015-02-01` not `2015-02-1`.

Reply:
Thank you very much.